src/index.css

```diff
@@ -11,3 +11,48 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 342px;
+  margin-bottom: 32px;
+}
```

Comment:
If we're writing this much custom CSS, we might consider using styled components instead and just rolling our own situation: https://styled-components.com/

Reply:
I am not opposed to styled components if we have to switch it up.